tests/phpunit/tests/query/dateQuery.php

```diff
@@ -20,34 +20,39 @@ public function setUp() {
 	}
 
 	public function _get_query_result( $args = array() ) {
-		$args = wp_parse_args( $args, array(
-			'post_status'            => 'any', // For the future post
-			'posts_per_page'         => '-1',  // To make sure results are accurate
-			'orderby'                => 'ID',  // Same order they were created
-			'order'                  => 'ASC',
-			'update_post_meta_cache' => false,
-			'update_post_term_cache' => false,
-		) );
+		$args = wp_parse_args(
+			$args,
+			array(
+				'post_status'            => 'any', // For the future post
+				'posts_per_page'         => '-1',  // To make sure results are accurate
+				'orderby'                => 'ID',  // Same order they were created
+				'order'                  => 'ASC',
+				'update_post_meta_cache' => false,
+				'update_post_term_cache' => false,
+			)
+		);
 
 		return $this->q->query( $args );
 	}
 
 	public function test_date_query_before_array() {
-		$p1 = self::factory()->post->create( array( 'post_date' => '2007-09-24 07:17:23',) );
-		$p2 = self::factory()->post->create( array( 'post_date' => '2008-03-29 07:17:23',) );
-		$p3 = self::factory()->post->create( array( 'post_date' => '2008-07-15 07:17:23',) );
-		$p4 = self::factory()->post->create( array( 'post_date' => '2009-06-11 07:17:23',) );
-
-		$posts = $this->_get_query_result( array(
-			'date_query' => array(
-				array(
-					'before' => array(
-						'year' => 2008,
-						'month' => 6,
+		$p1 = self::factory()->post->create( array( 'post_date' => '2007-09-24 07:17:23' ) );
```

Comment:
There are a bunch of these in this file, where the trailing comma is removed. Is this because it's one entry on one line?

Reply:
Yes, seems like that's the rule.

In the `WordPress.Arrays.CommaAfterArrayItem.CommaAfterLast` test you get:
"Comma not allowed after last value in single-line array declaration"
If a single value array declared on one line has a trailing comma.

Reply:
OK. Several rules depend on single or multi line context, making them fairly arbitrary.

Reply:
It is because they are only on one line.

If there was one rule in the standards from upstream that I'm not too keen on, it's the trailing comma on multiline arrays.

Reply:
I agree, but there are several rules I would remove if it were my choice, like having each array item on its own line, Yoda conditions, and so many blank lines everywhere.